### PR TITLE
Thread Details UI implementation 

### PR DESCRIPTION
**Pull request by richieoscar**

### CODE REVIEW

File: lib/ui/view/all_threads_view/threadDetail/thread_detail_viewmodel.dart [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+     
+    
+
+     void onMessageFieldTap(){
+       _isVisible = true;
+        notifyListeners();
+       
+     }
+     void pop() {
+    final navigationService = locator<NavigationService>();
```
Comment (Quwaysim) [inline lib/ui/view/all_threads_view/threadDetail/thread_detail_viewmodel.dart:32]: better make this a field

File: lib/ui/view/all_threads_view/threadDetail/hook_thread_detail.dart [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+                  Padding(
+                    padding: const EdgeInsets.all(12.0),
+                    child: Row(
+                      children: [
+                        Icon(
+                          Icons.pin,
+                          color: AppColors.greyishColor,
+                        ),
+                        SizedBox(width: 30),
+                        CustomText(text: "Pin to Conversation", fontSize: 15)
```
Comment (Quwaysim) [inline lib/ui/view/all_threads_view/threadDetail/hook_thread_detail.dart:430]: use one of the styles in styles.dart or add yours if it's not there

File: lib/ui/view/all_threads_view/threadDetail/hook_thread_detail.dart [hunk 3]
```diff
@@ -0,0 +1,442 @@
...
+                      ],
+                    ),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.all(15.0),
+                    child: Row(
+                      children: [
+                        Icon(Icons.copy_rounded, color: AppColors.greyishColor),
+                        SizedBox(width: 30),
+                        CustomText(text: "Copy Text", fontSize: 15)
```
Comment (Quwaysim) [inline lib/ui/view/all_threads_view/threadDetail/hook_thread_detail.dart:416]: use one of the styles in styles.dart or add yours if it's not there

File: lib/ui/view/all_threads_view/threadDetail/hook_thread_detail.dart [hunk 4]
```diff
@@ -0,0 +1,442 @@
...
+              child: Column(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.all(12.0),
+                    child: Row(
+                      children: [
+                        Icon(Icons.access_time_rounded,
+                            color: AppColors.borderColor),
+                        SizedBox(width: 30),
+                        CustomText(text: "Remind Me", fontSize: 15)
```
Comment (Quwaysim) [inline lib/ui/view/all_threads_view/threadDetail/hook_thread_detail.dart:350]: use one of the styles in styles.dart or add yours if it's not there..... DO THIS FOR ALL SIMILAR ISSUES

**Review by Quwaysim — CHANGES_REQUESTED** [review]
WHY is your app.dart empty?

**Comment by richieoscar** [comment]
> WHY is your app.dart empty?

Pulled incoming  changes... And found app.dart empty.. After merge

File: lib/ui/view/all_threads_view/threadDetail/hook_thread_detail.dart (hunk 4 above)
Comment (richieoscar) [reply lib/ui/view/all_threads_view/threadDetail/hook_thread_detail.dart:350]: Okay

**Pull request closed without merging**